packages/contracts/package.json

```diff
@@ -16,8 +16,9 @@
   "types": "dist/index",
   "license": "MIT",
   "scripts": {
-    "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
-    "build": "./scripts/build.sh",
+    "all": "yarn build && yarn test && yarn lint:fix && yarn lint",
+    "build": "yarn clean && ./scripts/build.sh && yarn copy-artifacts",
+    "copy-artifacts": "cp -R artifacts-ovm dist/ && cp -R deployments dist/",
```

Comment:
We can add these to the `files` field of `package.json` I think? This seems like it need not be a user-driven command and instead should be something as part of an automation of some other script.

Reply:
ahh yea - I just needed to add `deployments` to the `files` list. thanks